### ISSUE TITLE
How to replace existed scene content with my own fbx and opendrive?

### ISSUE THREAD

**Issue by wangludong** [carla-simulator/carla#3257]
Hi

I deleted most contents of a existed scene and tried to import my fbx and opendrive generated by RoadRunner,
but I can't generate routes markers.

Here are my steps:

RoadRunner version:R2020a
Carla version:0.9.8
Unreal Engine version:4.22.3

1.Make a simple segment of road in RoadRunner.
![xodr1](https://user-images.githubusercontent.com/167162/92085534-8ad4d680-edfb-11ea-94ce-1ac971e88c8f.PNG)

2.Export fbx and opendrive with Carla option.
![xodr2](https://user-images.githubusercontent.com/167162/92085687-bf489280-edfb-11ea-84dd-05f633fb5dde.PNG)

3.Import the fbx into UE4 and replace the xodr file with my own.
![xodr3](https://user-images.githubusercontent.com/167162/92085767-ddae8e00-edfb-11ea-96c2-a97d900e5413.PNG)

4.Drag a Open Drive Actor and drop it on my road, but no routes generated. I also tried to check "Generate Routes" Option in Details Panel and got no response either.
![xodr4](https://user-images.githubusercontent.com/167162/92086266-8957de00-edfc-11ea-8c17-2b35ba2630e7.PNG)

I hope someone could give me some suggestions.
Thanks a lot.

**Comment by thytless** [comment]
Hello! The "Generate Routes" Option in OpenDriveActor should work, did you also check the "Show Debug" option below?

**Comment by wangludong** [comment]
> Hello! The "Generate Routes" Option in OpenDriveActor should work, did you also check the "Show Debug" option below?

Hi
"Show Debug" option is checked in my project.
Actually, I followed this document [https://www.mathworks.com/help/roadrunner/ug/Exporting-to-CARLA.html](url).
Now I can run PythonAPI\examples\manual_control.py to drive the car manually or automatically.
But when I launched PythonAPI\examples\automatic_control.py, I got a number of errors below.
![automatic_error](https://user-images.githubusercontent.com/167162/92213386-07cc8280-eec6-11ea-8cd0-8706ec0bcfd3.JPG)

It seems like that the simulator can not localize the lane.
I totally don't know whether my operations are right.I think the route mark is supposed to be displayed in order to indicate where the route is.
